FSharp.Compiler.Tools.Nuget/build/FSharp.Compiler.Tools.props

```diff
@@ -14,20 +14,35 @@ WARNING:  You CAN MODIFY this file, doesnt matter if you are not knowledgeable a
 
   <PropertyGroup>
 
-    <!-- Default to netcoreapp fsc -->
-    <FscTaskAssemblyPath>$(MSBuildThisFileDirectory)netcoreapp1.0/FSharp.Build.dll</FscTaskAssemblyPath>
-    <FscToolPath>$(MSBuildThisFileDirectory)netcoreapp1.0</FscToolPath>
-    <FscToolExe>fsc.dll</FscToolExe>
-    <FscToolFullPath>$(MSBuildThisFileDirectory)netcoreapp1.0/fsc.dll</FscToolFullPath>
-
     <!-- specific netcoreapp1.0 fsc -->
     <FscTaskAssemblyPath_netcoreapp1_0>$(MSBuildThisFileDirectory)netcoreapp1.0/FSharp.Build.dll</FscTaskAssemblyPath_netcoreapp1_0>
     <FscToolFullPath_netcoreapp1_0>$(MSBuildThisFileDirectory)netcoreapp1.0/fsc.dll</FscToolFullPath_netcoreapp1_0>
+    <FscToolPath_netcoreapp1_0>$(MSBuildThisFileDirectory)netcoreapp1.0</FscToolPath_netcoreapp1_0>
```

Comment:
please remove that `FscToolPath_netcoreapp1_0`, i know is a good refactor, but i added `FscToolFullPath_netcoreapp1_0` because i had no other way to save paths at the time.
i prefer if you can duplicate the paths in following properies, instead of pollute msbuild with unused properties. these should be labelled also as private with `_` prefix because is something who will change.